lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -492,6 +526,14 @@ def fetch_ec2_metadata
       end
     end
 
+    def detect_cloudfunctions(attributes, common_labels)
+      return unless attributes.include?('gcf_region')
+      # Cloud Functions detected
+      @running_cloudfunctions = true
+      @gcf_region = fetch_gce_metadata('instance/attributes/gcf_region')
+      common_labels["#{CLOUDFUNCTIONS_SERVICE}/region"] = @gcf_region
```

Comment:
This gets included on entries that fail the 'if matchdata' test above on line 315 - seems like it should not be?